src/readToEarn.py

```diff
@@ -61,14 +61,15 @@ def completeReadToEarn(self,startingPoints):
         
         # json data to confirm an article is read
         json_data = {
-            'amount': 1,
-            'country': 'us',
-            'id': 1,
-            'type': 101,
-            'attributes': {
-                'offerid': 'ENUS_readarticle3_30points',
-                },
-            }
+            "amount": 1,
+            # "country": "us",

```

Comment:
I'm good with deleting commented-out code and relying on version control

Reply:
That looks good. I cannot test outside US. Also, I just did a PR to `develop` to add the daily check-in as well, and added this fix.